R/mc2_mthds.R

```diff
@@ -51,6 +52,16 @@
 #'   cval = |(cval - (mean cval for wllt = n and p)) / (sd cval for wllt = n and p)|.}
 #'   \item{sub1}{Center data around zero by subtracting the corrected response value (cval) from 1; 
 #'   1 - cval. Typically used if data was pre-normalized around 1 with responses decreasing to 0.}
+#'   }
+#' }
+#' 
+#' \subsection{Aggregation Methods}{
+#' \describe{
+#'  \item{agg.mean.rep.apid}{Aggregate technical replicates by taking the plate-wise mean per sample id and concentration index.}
+#'  \item{agg.median.rep.apid}{Aggregate technical replicates by taking the plate-wise median per sample id and concentration index.}
+#'  \item{agg.percent.rep.spid}{Use for binary data. Aggregate technical replicates as percentage by taking the sum of hits relative to total replicates per sample id and concentration index.}
+#'  \item{agg.percent.rep.spid.min1}{Use for binary data. Aggregate technical replicates as percentage by taking the sum of hits relative to total replicates per sample id and concentration index, where at least one hit is required.}
+#'   }
```

Comment:
1. For the the other methods, we have included calculations when possible; could calculations be included here? 
2. The other methods are written in this style, "Aggregate technical replicates by taking the plate-wise mean per sample id _(spid)_ and concentration index _(cndx)_."

Reply:
I think you could add a brief calculation similar to other calculations, for example ( sum(cval)/N )* 100 in the case of the aggregate percentage of replicates.  However, I will leave that up to you @Ashley-Ko and @madison-feshuk to make the final decision on what will be most helpful/what you want in that regard.